companyweb_base/security/security.xml

```diff
@@ -0,0 +1,18 @@
+<odoo>
+
+    <record model="ir.module.category" id="module_management">
+         <field name="name">Company Web Base</field>
```

Comment:
```suggestion
         <field name="name">Companyweb</field>
```

On companyweb.be, they use `Compayweb` in one word without first capital. Let's use that everywhere for consistency.

Reply:
I did this yesterday but haven't commited yet
grep "Company Web"
grep CompanyWeb
grep Company web